src/Microsoft.Identity.Web.OWIN/Microsoft.Identity.Web.OWIN.csproj

```diff
@@ -33,7 +33,7 @@
     <PackageReference Include="Microsoft.AspNet.Mvc" Version="5.2.7" />
     <PackageReference Include="Microsoft.AspNet.Web.Optimization" Version="1.1.3" />
     <PackageReference Include="Microsoft.AspNet.WebApi" Version="5.2.7" />
-    <PackageReference Include="Microsoft.IdentityModel.Protocols.WsFederation" Version="5.5.0" />
+    <PackageReference Include="Microsoft.IdentityModel.Protocols.WsFederation" Version="8.1.2" />
```

Comment:
web.config also has to be updated. Owin is a bit more complex. You will need to let VS Help and double click on the warning that should appear in the IDE.

Reply:
We actually aren't taking the WsFederation change